web/wp-content/themes/dctx/functions.php

```diff
@@ -237,12 +237,12 @@ function remove_empty_p($content) {
 	add_filter('the_content', 'remove_empty_p', 20, 1);
 }/*}}}*/
```

Comment:
:warning: **Warning**: This comment is 100% valid code; is this commented out code? (*Squiz.PHP.CommentedOutCode.Found*).